content/docs/intro/vs/serverless.md

```diff
@@ -13,14 +12,14 @@ aliases: ["/docs/reference/vs/serverless/"]
 
 The Serverless Framework is a tool that makes programming AWS Lambda, Azure Functions, and Google Cloud Functions
 easier, by removing much of the boilerplate out of the native cloud providers' development experiences. It generates
-the YAML and configuration required for functions, and simplifies uploading code packs to the cloud. It relies on
+the YAML and configuration required for functions, and simplifies uploading code packs to the cloud. The Serverless Framework relies on
 the cloud provider's APIs or templating solutions for any provisioning or management beyond the functions themselves.
 
-Pulumi is more holistic in its focus. In Pulumi, you write entire applications -- some parts of which may be serverless
+Pulumi is more holistic than the Serverless Framework in its focus. In [Pulumi] (https://www.pulumi.com/blog/tag/pulumi/), you write entire applications -- some parts of which may be serverless
```

Comment:
@97thFloor What is the reason for linking to our blog instead of the home page? Just curious

Reply:
Good catch @zchase looks like that was a mistake on our end, let's change that to the homepage.